src/borg/logger.py

```diff
@@ -149,8 +149,13 @@ def __logger(self):
                 if not configured:
                     raise Exception("tried to call a logger before setup_logging() was called")
                 self.__real_logger = logging.getLogger(self.__name)
+                if self.__name.startswith('borg.debug.') and self.__real_logger.level == logging.NOTSET:
+                    self.__real_logger.setLevel('WARNING')
```

Comment:
hmm, how is borg.debug.* special so it needs that treatment?
and other borg.something.else does not need this?